test/Microsoft.ML.Predictor.Tests/TestPredictors.cs

```diff
@@ -407,6 +407,24 @@ public void FastTreeBinaryClassificationTest()
             Done();
         }
 
+        [Fact]
+        [TestCategory("Binary")]
+        [TestCategory("LightGBM")]
+        public void LightGBMClassificationTest()
+        {
+            RunMTAThread(() =>
```

Comment:
>RunMTAThread [](start = 12, length = 12)

What is the purpose of this?

Reply:
Helper to create a background thread and wait for it to finish.

---
In reply to: [197461317](https://github.com/dotnet/machinelearning/pull/392#discussion_r197461317) [](ancestors = 197461317)

Reply:
Thanks. What I mean is, what is the purpose of your *call* of it? I was not asking what the method does. I am asking why you are calling it. What goes wrong if you just run it in the current thread?

---
In reply to: [197570408](https://github.com/dotnet/machinelearning/pull/392#discussion_r197570408) [](ancestors = 197570408,197461317)

Reply:
We could do that.

---
In reply to: [197575746](https://github.com/dotnet/machinelearning/pull/392#discussion_r197575746) [](ancestors = 197575746,197570408,197461317)